src/locale/locales/el/messages.po

```diff
@@ -0,0 +1,8024 @@
+
+#: src/view/com/post-thread/PostThreadItem.tsx:408
+msgid "{0, plural, one {repost} other {reposts}}"
+msgstr "{0, plural, one {repost} other {reposts}}"
+
+#: src/view/com/util/post-ctrls/PostCtrls.tsx:301
+msgid "{0, plural, one {Unlike (# like)} other {Unlike (# likes)}}"
+msgstr "{0, plural, one {Δεν μου αρέσει (# like)} other {Δεν μου αρέσει (# likes)}}"
+
+#: src/screens/Settings/Settings.tsx:414
+msgid "{0}"
+msgstr "{0}"
+
+#. Pattern: {wordValue} in tags
+#: src/components/dialogs/MutedWords.tsx:475
+msgid "{0} <0>in <1>tags</1></0>"
+msgstr "{0} <0>σε <1>ετικέτες</1></0>"
+
+#. Pattern: {wordValue} in text, tags
+#: src/components/dialogs/MutedWords.tsx:465
+msgid "{0} <0>in <1>text & tags</1></0>"
+msgstr "{0} <0>σε <1>κείμενο & ετικέτες</1></0>"
+
+#: src/screens/StarterPack/StarterPackLandingScreen.tsx:219
+msgid "{0} joined this week"
+msgstr "{0} εντάχθηκε αυτήν την εβδομάδα"
+
+#: src/view/com/util/post-embeds/VideoEmbedInner/web-controls/Scrubber.tsx:195
+msgid "{0} of {1}"
+msgstr "{0} από {1}"
+
+#: src/screens/StarterPack/StarterPackScreen.tsx:479
+msgid "{0} people have used this starter pack!"
+msgstr "{0} άτομα έχουν χρησιμοποιήσει αυτό το starter pack!"
+
+#: src/view/shell/bottom-bar/BottomBar.tsx:203
+msgid "{0} unread items"
+msgstr "{0} ανεπίλυτα στοιχεία"
+
+#: src/view/com/util/UserAvatar.tsx:435
+msgid "{0}'s avatar"
+msgstr "Η εικόνα προφίλ του {0}"
+
+#: src/screens/StarterPack/Wizard/StepDetails.tsx:67
+msgid "{0}'s favorite feeds and people - join me!"
+msgstr "Τα αγαπημένα feeds και άτομα του {0} - ενταχθείτε!"
+
+#: src/screens/StarterPack/Wizard/StepDetails.tsx:46
+msgid "{0}'s starter pack"
+msgstr "Το starter pack του {0}"
+
+#. How many days have passed, displayed in a narrow form
+#: src/lib/hooks/useTimeAgo.ts:158
+msgid "{0}d"
+msgstr "{0}ημ."
+
+#. How many hours have passed, displayed in a narrow form
+#: src/lib/hooks/useTimeAgo.ts:148
+msgid "{0}h"
+msgstr "{0}ωρ."
+
+#. How many minutes have passed, displayed in a narrow form
+#: src/lib/hooks/useTimeAgo.ts:138
+msgid "{0}m"
+msgstr "{0}λ."
+
+#. How many months have passed, displayed in a narrow form
+#: src/lib/hooks/useTimeAgo.ts:169
+msgid "{0}mo"
+msgstr "{0}μην."
+
+#. How many seconds have passed, displayed in a narrow form
+#: src/lib/hooks/useTimeAgo.ts:128
+msgid "{0}s"
+msgstr "{0}δευτ."
+
+#: src/view/shell/bottom-bar/BottomBarWeb.tsx:252
+msgid "{badge} unread items"
+msgstr "{badge} ανεπίλυτα στοιχεία"
+
+#: src/components/LabelingServiceCard/index.tsx:96
+msgid "{count, plural, one {Liked by # user} other {Liked by # users}}"
+msgstr "{count, plural, one {Μου άρεσε από # χρήστη} other {Μου άρεσε από # χρήστες}}"
+
+#: src/view/shell/desktop/LeftNav.tsx:223
+msgid "{count} unread items"
+msgstr "{count} ανεπίλυτα στοιχεία"
+
+#: src/lib/generate-starterpack.ts:108
+#: src/screens/StarterPack/Wizard/index.tsx:183
+msgid "{displayName}'s Starter Pack"
+msgstr "Το Starter Pack του {displayName}"
```

Comment:
Further down this was translated, maybe it should be kept like that here as well. What do you think?

Starter Pack -> Αρχικό Πακέτο

Reply:
θα πρότεινα `πακέτο έναρξης`

Reply:
Νομίζω ότι πλέον χρησιμοπείται το Starter Pack και από τους Έλληνες χρήστες. Νομίζω θα χάνονται άμα το μεταφράσουμε.
Προτείνω να το αφήσουμε Starter Pack